### PR TITLE
add auto-update config & update package.json of highcharts

### PR DESCRIPTION
**Pull request by Amomo**
Hi @LeaYeh , 
For #5194 and #5500 , I update the license & repository field in first commit.

highcharts use `"Creative Commons Attribution-NonCommercial 3.0 License"`,
You can find this info from http://shop.highsoft.com/highcharts.html
And, according to [SPDX License List](https://spdx.org/licenses/), it could be noted as `"CC-BY-NC-3.0"`

For #5525 , I add git auto-update config in the second commit.
repo: https://github.com/highslide-software/highcharts-release

download page: http://www.highcharts.com/download
official release: http://code.highcharts.com/zips/Highcharts-4.1.8.zip

Would you mind checking this PR for me? Thank you~ :-)

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Where is your check list?

**Comment by Amomo** [comment]
- [ ] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity : **Watch `274`, Star `4,103`, Fork `1,496`**
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.

**Comment by PeterDaveHello** [comment]
Why don't you use the release repo?
https://github.com/highslide-software/highcharts-release

**Comment by Amomo** [comment]
Oh! I am using a wrong repo! Let me fix it later.

**Comment by LeaYeh** [comment]
should we also point `filename` to min file?

**Comment by Amomo** [comment]
@LeaYeh `highcharts.js` is a minified file, they use `highcharts.src.js` stand for source file.
BTW, I have rebased my PR.

File: ajax/libs/highcharts/package.json [hunk 1]
```diff
@@ -16,5 +16,18 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/highslide-software/highcharts.com.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/highslide-software/highcharts-release.git",
+    "basePath": "",
+    "files": [
+      "**/*.+(css|js|png)"
```
Comment (LeaYeh) [inline ajax/libs/highcharts/package.json]: maybe we dont need `dot` , do we?

Comment (Amomo) [inline ajax/libs/highcharts/package.json]: I think we should use the dot sign to confirm the ending is a file extension.

Comment (LeaYeh) [inline ajax/libs/highcharts/package.json]: hmmm... your right. i got a mistake

**Comment by Amomo** [comment]
@LeaYeh Next time, if you think a PR is fine, could you tag Peter and call him to check? 😃
And, please assign the PR to him. Thank you~

**Comment by LeaYeh** [comment]
@PeterDaveHello could check this pr?

**Pull request merged**